### PR TITLE
Create draft of public addresses.md

### PR DESCRIPTION
**Pull request by MCM-Mike**
First draft of the new GRIN community council public BTC address.

### CODE REVIEW

**Review by Anynomouss — APPROVED** [review]

File: addresses.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Contact
+Any questions or concerns about authenticity can be raised publicly in the Grin community via [keybase](https://keybase.io/team/grincoin) 
+**tbd**
+
+---
+
+## List of addresses
+
+### Grin General Fund
```
Comment (marekyggdrasil) [inline addresses.md:33]: To me wording "general fund" is confusing with [grin general fund](https://grin.mw/fund) which (correct me if I'm wrong) is different than fund used by the Community Council.

Comment (MCM-Mike) [reply addresses.md]: changed it to grin community council fund

**Review by MCM-Mike — COMMENTED** [review]
changed naming to GRIN community council fund

**Review by davidtavarez — APPROVED** [review]
Setting up a Grin Community Council Grin node with donation address is a topic of the 06 July 2021 agenda.

**Review by hendi — APPROVED** [review]

**Comment by davidtavarez** [comment]
@MCM-Mike feel free of merging this.

**Review by MCM-Mike — COMMENTED** [review]
final changes and formatting.

**Pull request merged**